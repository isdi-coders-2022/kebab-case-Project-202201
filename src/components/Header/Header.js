import styled from "styled-components";

const HeaderBar = styled.div`
  height: 60px;
  width: 100%;
  background: #fff;
  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
  position: absolute;
  display: flex;
  justify-content: center;
  flex-direction: column;
  align-items: center;
`;

const StreamingImg = styled.img`
  position: absolute;
  width: 87px;
  height: 16px;
  left: 70px;
  top: 3px;
`;

const ShawarmaImg = styled.img`
  position: absolute;
  width: 186px;
  height: 31px;
  left: 102px;
  top: 19px;
`;

const LogoImg = styled.img`
  width: 38px;
  height: 47px;
  padding-left: 15px;
  align-self: flex-start;
`;

const Header = () => {
  return (
    <>
      <HeaderBar>
        <LogoImg src={"img/logo.png"}></LogoImg>
        <StreamingImg src={"img/streaminglight.png"}></StreamingImg>
        <ShawarmaImg src={"img/shawarmalight.png"}></ShawarmaImg>
      </HeaderBar>
    </>
  );
};

export default Header;
