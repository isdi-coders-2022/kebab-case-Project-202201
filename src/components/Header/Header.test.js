import { render, screen } from "@testing-library/react";
import Header from "./Header";

describe("Given Header component", () => {
  describe("When it gets a Alt text /logo shawarma/", () => {
    test("then it should found an image in Header with alt /logo shawarma/", () => {
      render(<Header />);

      const streamingImg = screen.queryByAltText("logo shawarma");

      expect(streamingImg).toHaveAttribute("alt", "logo shawarma");
    });
  });
});
