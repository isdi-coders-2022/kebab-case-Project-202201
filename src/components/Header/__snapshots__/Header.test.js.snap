// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given Header component When it get a const App that creates Header then it should create and compare a snapshot with Header component 1`] = `
<header
  className="sc-gsDKAQ iagqhg"
>
  <button
    className="sc-bdvvtL feheQX"
    type="button"
  >
    <img
      alt="logo shawarma"
      src="img/logo.png"
    />
  </button>
  <img
    alt="streaming header title"
    className="sc-dkPtRN eSHNya"
    src="img/streaminglight.png"
  />
  <img
    alt="streaming header title"
    className="sc-hKwDye fVvAZv"
    src="img/shawarmalight.png"
  />
</header>
`;
